src/inc/msquic_posix.h

```diff
@@ -79,7 +79,7 @@ QUIC_INLINE ENUMTYPE &operator ^= (ENUMTYPE &a, ENUMTYPE b) throw() { return (EN
 
 #define QUIC_API
 #define QUIC_MAIN_EXPORT
-#define QUIC_STATUS                     unsigned int
+#define QUIC_STATUS                     int
```

Comment:
Thinking more about the `unsigned int` -> `int` change, why do we need it?

I am starting to have concerns about compile time breaking changes.

Reply:
That's a fail point. I've reverted the type changes and simply have the casting